src/conversations/UserGroup.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+import { Avatar, Touchable } from '../common';
+import { BRAND_COLOR } from '../common/styles';
+
+const styles = StyleSheet.create({
+  row: {
+    height: 44,
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: 8,
+  },
+  selectedRow: {
+    backgroundColor: BRAND_COLOR,
+  },
+  touchTarget: {
+  },
+  text: {
+    flex: 1,
+    marginLeft: 10,
+    marginRight: 10,
+  },
+  selectedText: {
+    color: 'white',
+  }
+});
+
+export default class UserGroup extends Component {
+
+  props: {
+    email: string,
+    fullName: string,
+    avatarUrl: string,
+    status: string,
+    isSelected: boolean,
+    onPress: () => void,
+  }
+
+  handlePress = () =>
+    this.props.onPress(this.props.email);
+
+  render() {
+    const { allNames, isSelected } = this.props;
+
+    return (
```

Comment:
The formatting seems a bit messed up when I look in the simulator. If you have a conversation with many users, the list of users overflows into bordering cells.